R/test-files.R

```diff
@@ -109,12 +109,19 @@ test_dir <- function(path,
 #' @inheritSection test_dir Environments
 #' @param path Path to file.
 #' @param ... Additional parameters passed on to `test_dir()`
+#' @param label Optionally, supply a string here to run only tests that
+#'   with this exact label.
 #' @export
 #' @examples
 #' path <- testthat_example("success")
 #' test_file(path)
+#' test_file(path, label = "some tests have warnings")
 #' test_file(path, reporter = "minimal")
-test_file <- function(path, reporter = default_compact_reporter(), package = NULL, ...) {
+test_file <- function(path,
+                      reporter = default_compact_reporter(),
+                      label = NULL,
```

Comment:
I think I'm sort of surprised that `label` isn't being added after `...`.

Reply:
I'm not sure how we should remediate functions where `...` is in the wrong place so I'm sticking with the existing convention.